python/pyspark/sql/column.py

```diff
@@ -187,18 +187,26 @@ class Column:
     """
     A column in a DataFrame.
 
-    :class:`Column` instances can be created by::
+    .. versionadded:: 1.3.0
 
-        # 1. Select a column out of a DataFrame
+    Examples
+    --------
+    :class:`Column` instances can be created by::
 
-        df.colName
-        df["colName"]
+    >>> df = spark.createDataFrame(
+    ...      [(2, "Alice"), (5, "Bob")], ["age", "name"])
 
-        # 2. Create from an expression
-        df.colName + 1
-        1 / df.colName
+    1. Select a column out of a DataFrame
+    >>> df.name
+    Column<'name'>
+    >>> df["name"]
+    Column<'name'>
 
-    .. versionadded:: 1.3.0
+    2. Create from an expression
```

Comment:
```suggestion
    Create from an expression

```

Reply:
Otherwise, I think the documentation rendering is broken IIRC

Reply:
Thanks, updated in latest commit. Let's see result.